__tests__/config.ts

```diff
@@ -0,0 +1,62 @@
+import { getConfig } from '../src/config';
+import { readJson } from '../src/fs';
+
+jest.mock('../src/fs');
```

Comment:
Is it possible to not mock the file system and _actually_ load config files.  looking at [`getConfig`](https://github.com/OpenSourceRaidGuild/unimported/blob/master/src/config.ts#L18), it's not doing much other than calling the `fs` module so the test becomes a self fulfilling prophecy (you returned the mock I told you to return).

Perhaps it can be reworked as an [integration test that writes a config file as part of a test project](https://github.com/OpenSourceRaidGuild/unimported/blob/master/__tests__/cli.ts#L40-L50)?